app/app.js

```diff
@@ -1,12 +1,29 @@
 const express = require("express");
-const { getTopics } = require("./controllers/controllers");
+const { getTopics, getArticles } = require("./controllers/controllers");
 
 const app = express();
 app.use(express.json());
```

Comment:
app.use(express.json()) is not needed here, and thus, it should be deleted & only re-introduced once you actually need to use it